WcaOnRails/db/migrate/20231006072155_remove_probation_from_teams.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveProbationFromTeams < ActiveRecord::Migration[7.0]
+  def change
+    Team.c_find_by_friendly_id!('probation').destroy
```

Comment:
Be careful with `destroy` invocations. In your PR description, you wrote that this PR won't remove any rows from `team_members`, but actually it is possible that Rails "magically" (i.e. by virtue of the fact that _we defined_ in our code that there is an association between Team and TeamMember) realizes "oh, the parent team doesn't exist anymore, so all other rows can be cleaned up".

Reply:
I think you are right. I just now checked my database and couldn't find the entry in `team_members` table. In that case, can I do this as well manually instead of going through migration?

Reply:
Yes, absolutely!